server/frontend/src/components/Bugs/PublicationForm.vue

```diff
@@ -780,12 +780,15 @@ export default {
           let content = this.testCaseContent;
           // If the testcase is binary we need to download it first
           if (this.entry.testcase_isbinary) {
-            content = await api.retrieveCrashTestCase(this.entry.id);
+            const resp = await api.retrieveCrashTestCaseBinary(this.entry.id);
+            content = new Uint8Array(resp.data);
```

Comment:
You should directly do that step in `retrieveCrashTestCaseBinary` so the internal API stays consistent (we get usable payloads from the `api.js` module)